tailcfg/tailcfg.go

```diff
@@ -1055,6 +1055,11 @@ type Debug struct {
 	// The client can (and should) limit the value (such as 5
 	// minutes).
 	SleepSeconds float64 `json:",omitempty"`
+
+	// RandomizeClientPort is whether magicsock should UDP bind to
+	// :0 to get a random local port, ignoring any configured
```

Comment:
Right now tailscaled has a `--port` which defaults to 0, meaning "we pick", and we pick 41641. This overrides it so we pick a random port. What if a user sets `--port 9999`? Does this override that?

Reply:
Yes, that's why I wrote "ignoring any configured"

Reply:
> Right now tailscaled has a --port which defaults to 0, meaning "we pick", and we pick 41641

No, the systemd config we use passes --port=41641 (see https://github.com/tailscale/tailscale/blob/main/cmd/tailscaled/tailscaled.service and https://github.com/tailscale/tailscale/blob/main/cmd/tailscaled/tailscaled.defaults)

Reply:
OK, then this isn't a feature we would want to use widely, definitely not be default for new users. But maybe that's fine because we want to do the double-port thing in the future or work around these NATs some other way?